packages/flutter_tools/lib/runner.dart

```diff
@@ -61,6 +62,44 @@ Future<int> run(
     StackTrace? firstStackTrace;
     return runZoned<Future<int>>(() async {
       try {
+        // Ensure that the consent message has been displayed
```

Comment:
Can you add a test for this to https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/general.shard/runner/runner_test.dart